themes/nav-community-v1/layouts/partials/footer.html

```diff
@@ -15,7 +15,8 @@ <h3>Important Links</h3>
                             <li><a href="https://navhub.org/news/" target="_blank">Latest News</a></li>
                             <li><a href="https://info.navcoin.org/faq/" target="_blank">FAQ</a></li>
                             <li><a href="https://navhub.org/roadmap/" target="_blank">Community Roadmap</a></li>
-                            <li><a href="https://chainz.cryptoid.info/nav/" target="_blank">Block Explorer</a></li>
+                            <li><a href="https://chainz.cryptoid.info/nav/" target="_blank">Block Explorer - Cryptoid</a></li>
+                            <li><a href="https://www.navexplorer.com" target="_blank">Block Explorer - NavExplorer</a></li>
```

Comment:
Change `Cryptoid` to `CryptoId` and you've got a deal.

Reply:
Or rename to CryptoID as written on their website, and aligned with navhub.org PR (https://github.com/NAVCoin/NavHub/pull/124)?